### ISSUE TITLE
Basic UI does not update icons of read only switches

### ISSUE THREAD

**Issue by wborn** [eclipse/smarthome#2278]
I've done some more testing with the functionality provided by #2030 and it seems that icons of read only switches in the Basic UI are not properly updated. I typically use these for representing the state of motion/smoke/CO sensors.

You can easily reproduce this in the demo by replacing the `Group demo` in `demo.sitemap` with

```
		Text label="Group Demo" icon="firstfloor" {
			Switch item=Lights mappings=[OFF="All Off"]
			Group item=Heating
			Group item=Windows
			Text item=Temperature
			Switch item=Light_C_Corridor_Ceiling
			Text item=Light_C_Corridor_Ceiling
		}
```

and replacing the definition of `Light_C_Corridor_Ceiling` in `demo.items` with:
```
Switch Light_C_Corridor_Ceiling     "Ceiling [%s]" <switch> (gC, Lights)
```

When you then switch `Light_C_Corridor_Ceiling` or `Lights mappings`, the icon of `Switch item=Light_C_Corridor_Ceiling` is properly updated. 

However the icon of `Text item=Light_C_Corridor_Ceiling` is not.

Only after reloading the page it will again represent the right state.

Another way to reproduce it is to open `Group Demo > Open windows`. And issue the following commands in the console:

```
smarthome:update Window_GF_Frontdoor OPEN
smarthome:update Window_GF_Kitchen OPEN
smarthome:update Window_GF_Living OPEN
smarthome:update Window_GF_Toilet OPEN
smarthome:update Window_FF_Bath OPEN
smarthome:update Window_FF_Bed OPEN
smarthome:update Window_FF_Office_Window OPEN
smarthome:update Window_FF_Office_Door OPEN
smarthome:update Garage_Door OPEN
```

Now all window icons should be open. But they only are after reloading the page. The same way windows should be closed after issuing:

```
smarthome:update Window_GF_Frontdoor CLOSED
smarthome:update Window_GF_Kitchen CLOSED
smarthome:update Window_GF_Living CLOSED
smarthome:update Window_GF_Toilet CLOSED
smarthome:update Window_FF_Bath CLOSED
smarthome:update Window_FF_Bed CLOSED
smarthome:update Window_FF_Office_Window CLOSED
smarthome:update Window_FF_Office_Door CLOSED
smarthome:update Garage_Door CLOSED
```

**Comment by kaikreuzer** [comment]
I can reproduce this. The event is correctly sent (as can be seen by the fact that the label texts are updated correctly), so I assume that is something for @resetnow to fix.

**Comment by wborn** [comment]
Thanks a lot @resetnow. It seems to be fixed with #2294 :-) The visibility changes also works well!
